ui/raidboss/data/03-hw/dungeon/gubal_library_hard.js

```diff
@@ -191,8 +201,8 @@
     },
     {
       id: 'Gubal Hard Ecliptic Meteor',
-      regex: /14:195D:Behemoth Ward starts using Ecliptic Meteor/,
-      regexFr: /14:195D:Béhémoth [Cc]onjuré starts using Météore [éÉ]cliptique/,
+      regex: / 14:195D:Behemoth Ward starts using Ecliptic Meteor/,
```

Comment:
Can you not add spaces to other lines? If the `starts using` conversion is also in progress, this will cause conflicts.  I was hoping this patch would only touch `1A` lines.

Reply:
sry for that, reverted the change

Reply:
Are they reverted? It does not look like it from the diff, but maybe I'm confused?